### PR TITLE
Refactoring: replace "export" and "import" with the single "clone_vm" method

### PR DESCRIPTION
**Pull request by legal90**
This makes code a little bit clearer and allows to implement some other `clone` features further. Actually, we perform `prlctl clone` on both `import` and `export` actions.

cc:\ @racktear, @Gray-Wind

### CODE REVIEW

File: test/unit/support/shared/pd_driver_examples.rb [hunk 1]
```diff
@@ -115,17 +115,28 @@
...
+    src_name = "Some_Template_Name"
+    dst_uuid = "1234-some-template-uuid-5678"
 
-    it "exports VM to template" do
+    it "clones VM to the new one" do
+      subprocess.should_receive(:execute).
+        with("prlctl", "clone", uuid, "--name", an_instance_of(String),
+             an_instance_of(Hash)).
+        and_return(subprocess_result(exit_code: 0))
+      subject.clone_vm(uuid, tpl_name).
```
Comment (Gray-Wind) [inline test/unit/support/shared/pd_driver_examples.rb]: tpl_name → src_name

File: test/unit/support/shared/pd_driver_examples.rb [hunk 2]
```diff
@@ -115,17 +115,28 @@
...
+    end
+
+    it "clones VM to template" do
       subprocess.should_receive(:execute).
         with("prlctl", "clone", uuid, "--name", an_instance_of(String),
              "--template", "--dst", an_instance_of(String),
              an_instance_of(Hash)).
         and_return(subprocess_result(exit_code: 0))
-      subject.export("/path/to/template", tpl_name).should == tpl_uuid
+      subject.clone_vm(uuid, tpl_name,
```
Comment (Gray-Wind) [inline test/unit/support/shared/pd_driver_examples.rb]: tpl_name → src_name

File: lib/vagrant-parallels/driver/pd_8.rb [hunk 3]
```diff
@@ -152,31 +152,18 @@ def enable_adapters(adapters)
...
 
-        def import(tpl_name)
-          vm_name = "#{tpl_name}_#{(Time.now.to_f * 1000.0).to_i}_#{rand(100000)}"
+        def clone_vm(src_name, dst_name, options={})
+          args = ['clone', src_name, '--name', dst_name]
+          args << '--template' if options[:template]
+          args.concat(['--dst', options[:dst]]) if options[:dst]
 
-          execute_prlctl('clone', tpl_name, '--name', vm_name) do |type, data|
+          execute_prlctl(*args) do |type, data|
```
Comment (Gray-Wind) [inline lib/vagrant-parallels/driver/pd_8.rb]: type → _

**Comment by legal90** [comment]
I've rebased it according to recommendations

**Pull request merged**